tests/helper.py

```diff
@@ -47,6 +64,14 @@ def find_ids(data, data_table, id_table):
             ids.append(id_table[i])
     return ids
 
+
+def to_ary(outfile, a, k):
```

Comment:
the api should probably be `to_ary(a, k)` and check `which_outfile()` in the conditional statement